app/renderer/js/components/webview.js

```diff
@@ -165,7 +165,13 @@ class WebView extends BaseComponent {
 	focus() {
 		// focus Webview and it's contents when Window regain focus.
 		const webContents = this.$el.getWebContents();
-		if (webContents && !webContents.isFocused()) {
+		// HACK: webContents.isFocused() seems to be true even without the element
+		// being in focus. So, we check against `document.activeElement`.
+		if (!(this.$el === document.activeElement)) {
```

Comment:
Why not use simple !== condition ? Also, we need for webContents here, because we are focusing on it later on. Rest of it seems fine. LGTM ! :)

Reply:
Thanks @vsvipul  for the review. I've pushed changes addressing your comments